core/tasks/starts/start_flow.go

```diff
@@ -136,7 +136,13 @@ func createFlowStartBatches(ctx context.Context, rt *runtime.Runtime, start *mod
 			batchTask = &StartFlowBatchTask{FlowStartBatch: batch}
 		}
 
-		err = tasks.Queue(rc, q, start.OrgID, batchTask, queue.DefaultPriority)
+		// if this is a big multi batch blast, give it low priority
+		priority := queue.DefaultPriority
+		if len(idBatches) > 0 {
```

Comment:
we always have  at least a batch, should we check the length is greater than 1?

Reply:
ha doh, yes